homeassistant/components/tuya/sensor.py

```diff
@@ -741,7 +742,7 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
     # https://developer.tuya.com/en/docs/iot/dlq?id=Kb0kidk9enyh8
     "dlq": (
         TuyaSensorEntityDescription(
-            key=DPCode.TOTAL_FORWARD_ENERGY,
+            key=DPCode.FORWARD_ENERGY_TOTAL,
```

Comment:
This looks to be a different field of data. Wouldn't this be a breaking change?

Reply:
@elupus This PR is a minor change of the "dlq category", our customer want to show currency, power and voltage values for circuit breaker in HA.  "TOTAL_FORWARD_ENERGY" is DP 1.0, "FORWARD_ENERGY_TOTAL" is DP 2.0 in Tuya IoT Platform DP rule definition. We just push a new commit to support both of them.  

Please check if this PR changes is reasonable, if it's considered to be a "Breaking change", we can close this PR and resubmit again. Thanks!

Reply:
I don't know what dlp category means, so I don't know if the old category is the same data as the old or not. So this is something somebody else need to look at.

Reply:
The "dlq" represents "circuit breakers" in Tuya's Standard status set documentation. The "TOTAL_FORWARD_ENERGY"  code is listed in the following doc, and the new "FORWARD_ENERGY_TOTAL" code will be added to the doc soon: https://developer.tuya.com/en/docs/iot/s?id=Kb0kikd1lya17.

Reply:
> We just push a new commit to support both of them.

That sounds off, as the old one is also still there; resulting in users ending up with duplicate sensors?